dev-client/app.config.ts

```diff
@@ -0,0 +1,71 @@
+import {ExpoConfig, ConfigContext} from 'expo/config';
```

Comment:
issue: add copyright notice.

Reply:
i don't understand which files need to have copyright notices and which don't. e.g. why don't we have one in `metro.config.js`? or in our yaml files?

Reply:
`metro.config.js` should have one too. That was an oversight. I hadn't thought about the YAML files before, just source code. We can add notices there too.